docs/blog/e18e.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: e18e (Ecosystem Performance) - A new community initiative
+title: e18e (Ecosystem Performance), a new community initiative
```

Comment:
this one isn't a grammatical thing, it is rather just following the `{title} - {subtitle}` format of titles which many blogs follow

if this was in the page, i'd probably agree a comma is fine. in the title in particular, though, it is a convention/consistency thing. the loss of grammatical accuracy is fine in this case for that reason